### ISSUE TITLE
Support for multiple file analysis (Entire project directory)

### ISSUE THREAD

**Issue by rohitcoder** [tree-sitter/tree-sitter-bash#266]
Hi, 

I was playing with tree-sitter playground and it works good with single file analysis, I was thinking if there is a module or any future plans for adding analysis of entire source code repo, kind of telling me entire path of a function. Source to Sink for Security analysis.

Let me know if we already have something like this, or you can also recommend me some approach how to move ahead for achieving this.

**Comment by ObserverOfTime** [comment]
Tree-sitter is not LSP.

**Comment by amaanq** [comment]
yeah, like mentioned above ^^ you can build something on *top* of tree-sitter to accomplish that, but tree-sitter does not provide this functionality.
